client/src/components/hooks/useStakeDeploys.js

```diff
@@ -1,7 +1,7 @@
 import { useEffect } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
 import { moteToCspr } from '../../helpers/balance';
-import { getPendingStakes } from '../../selectors/stake';
+import { getPendingStakes, getStakesHistory } from '../../selectors/stake';
```

Comment:
Minor: while updating new features, we can keep an eye on these legacy imports by gradually changing to alias import `@cd/selectors/....` .